lib/cisco_node_utils/interface.rb

```diff
@@ -132,6 +152,9 @@ def self.capabilities(intf, mode=:hash)
     def create
       feature_vlan_set(true) if @name[/(vlan|bdi)/i]
       config_set('interface', 'create', name: @name)
+      if @name[/ethernet/] && !@name[/ethernet.*\.\d+/]
```

Comment:
This seems odd. Are we changing a property just to make it non-default even though the manifest may not specify it?

Reply:
I added this so that creating an interface for physical interfaces and setting properties to default values is idempotent.  I needed at least one property to be set to a non-default value under the physical interface for it to be in a non-default state.  Without this if the ethernet interface is in the `absent` state (default state) and we try and create an interface via `ensure => present` and/or set properties under the interface to `default` values it will not be idempotent.

Reply:
This seemed like a fairly harmless way to allow us to manage physical interfaces and maintain idempotence.